src/compiler/checker.ts

```diff
@@ -1473,8 +1476,6 @@ namespace ts {
             return undefined;
         }
 
-        // This is for caching the result of getSymbolDisplayBuilder. Do not access directly.
-        let _displayBuilder: SymbolDisplayBuilder;

```

Comment:
No one is going to want that amount of verbosity to disable things, especially one liners